packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1218,6 +1219,18 @@ export const AVAILABLE_RESOLUTIONS = {
       },
     },
   },
+  '540p': {
```

Comment:
`AVAILABLE_RESOLUTIONS` are not used at all in SDK code and not used by the web app. I think it could be completely removed as it's something left behind from previous SDK versions from the times before SDK was using webrtc-core lib. @arun3528 can you confirm, if we can remove it?